docs/design/2023-02-21-tiflash-supports-stale-read.md

```diff
@@ -0,0 +1,31 @@
+# TiFlash supports stale read
+
+- Author: [hehechen](https://github.com/hehechen)
+
+## Introduction
+
+This proposal aims to make TiFlash support Stale Read.
+
+## Motivation or Background
+
+Currently TiKV has already supported [stale read](https://docs.pingcap.com/tidb/dev/stale-read), but TiFlash does not.
+There are two benefits of TiFlash to support stale read:
+- Eliminates the read/wait index duration during read processing. The small queries should run faster.
+- Avoid calling read_index API of TiKV leaders, to reduces the impact of OLTP workloads.
+
+## Goals
+
+1. TiFlash supports stale read, including [AS OF TIMESTAMP](https://docs.pingcap.com/tidb/dev/as-of-timestamp), [TIDB_BOUNDED_STALENES](https://docs.pingcap.com/tidb/dev/as-of-timestamp#syntax), [tidb_read_staleness](https://docs.pingcap.com/tidb/dev/tidb-read-staleness), [tidb_external_ts](https://docs.pingcap.com/tidb/dev/tidb-external-ts).
+2. Since TiFlash MPP may involve the computation of all TiFlash nodes, it does not support [using stale read to read TiFlash data nearby](https://docs.pingcap.com/tidb/dev/three-dc-local-read).
```

Comment:
```suggestion
2. Since TiFlash MPP may involve the computation among all TiFlash nodes, supporting [using stale read to read TiFlash data nearby](https://docs.pingcap.com/tidb/dev/three-dc-local-read) is not the goal of this RFC.
```